hudi-hadoop-common/src/main/java/org/apache/hudi/io/storage/HoodieHBaseAvroHFileReader.java

```diff
@@ -95,16 +96,29 @@ public HoodieHBaseAvroHFileReader(StorageConfiguration<?> storageConf, StoragePa
     this(path, HoodieStorageUtils.getStorage(path, storageConf), storageConf, cacheConfig, Option.empty());
   }
 
+  public HoodieHBaseAvroHFileReader(StorageConfiguration<?> storageConf, StoragePath path, Option<Schema> schemaOpt) throws IOException {
+    this(storageConf, path, new CacheConfig(storageConf.unwrapAs(Configuration.class)), new HoodieHadoopStorage(path, storageConf), schemaOpt);
+  }
+
   public HoodieHBaseAvroHFileReader(StorageConfiguration<?> storageConf, StoragePath path, CacheConfig cacheConfig,
                                     HoodieStorage storage, Option<Schema> schemaOpt) throws IOException {
     this(path, storage, storageConf, cacheConfig, schemaOpt);
   }
 
+  public HoodieHBaseAvroHFileReader(StorageConfiguration<?> storageConf, StoragePath path, HoodieStorage storage,
```

Comment:
Do we still need other constructors?  There are many.

Reply:
reduced